src/components/IframeComponent/loader.vue

```diff
@@ -1,128 +1,103 @@
 <template>
-    <div class="container">
-      <div class="row">
-        <div class="col-xl-4 pt-5">
-          <div class="placeholder wave">
-            <div class="project-block">
-              <!-- <div class="sidebar"></div> -->
-              <div class="workflow">
-                <section  class="loader">
-                  <span class="loader-text pi pi-spin pi-spinner"></span>
-                  <span class="loader-text">
-                    Experience are getting started.
-                  </span>
-                  
-                </section>
-              </div>
+  <div class="container">
+    <div class="row">
+      <div class="col-xl-4 pt-5">
+        <div class="placeholder wave">
+          <div class="project-block">
+            <!-- <div class="sidebar"></div> -->
+            <div class="workflow">
+              <section class="loader">
+                <span class="loader-text"> Experience are getting ready. </span>
+                <span class="loader-text pi pi-spin pi-spinner"></span>
+              </section>
             </div>
           </div>
         </div>
       </div>
     </div>
-  </template>
-  <script>
-    export default {
-      name: "Loadingpage",   
-     
+  </div>
+</template>
+<script>
+  export default {
+    name: "Loadingpage",
```

Comment:
Can you please follow camel case for this name.

Reply:
Done